### PR TITLE
Demonstration of Arrays.fill, Arrays.setAll, Arrays.parallelSetAll methods

### PR DESCRIPTION
**Pull request by jsayev**
- Wrapped builtin fill, setAll, parallelSetAll static methods from java.util.Arrays helper class inside custom methods;
- Demonstrated the most generic version of all three methods stated above;
- Tried to document all overloaded versions of methods shown above;
- Included extra showcase for writing lambda functions, instantiating anonymous classes;
**Note: initial commits had mistakes in them. Tried to fix. Failed few times.
Note to self: If you want to provide link to specific issue it is wise to do like "#N or Issue #N" where N stands for the number for the issue**

### CODE REVIEW

File: src/main/java/com/epam/prd/jjop/helpers/Arrays.java [hunk 1]
```diff
@@ -0,0 +1,210 @@
+package com.epam.prd.jjop.helpers;
```
Comment (marcin-ogorzalek-EPAM) [inline src/main/java/com/epam/prd/jjop/helpers/Arrays.java:1]: Lack of inner package dedicated for this task.

File: src/main/java/com/epam/prd/jjop/helpers/Arrays.java [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+import java.util.function.IntFunction;
+import java.util.function.IntToDoubleFunction;
+import java.util.function.IntToLongFunction;
+import java.util.function.IntUnaryOperator;
+
+import static java.util.Arrays.*;
+
+/**
+ * Demonstrates fill and setAll static methods from {@link java.util.Arrays} helper class by using them inside documented custom methods
```
Comment (marcin-ogorzalek-EPAM) [inline src/main/java/com/epam/prd/jjop/helpers/Arrays.java:11]:
```suggestion
 * Demonstrates {@code fill} and {@code setAll} static methods from {@link java.util.Arrays} helper class by using them inside documented custom methods
```

**Comment by marcin-ogorzalek-EPAM** [comment]
Very nice examples with quite detailed javadocs.

**Comment by jsayev** [comment]
Solved all issues stated by @marcin-ogorzalek-EPAM.

**Comment by marcin-ogorzalek-EPAM** [comment]
@jsayev so why is this closed?

**Comment by jsayev** [comment]
I thought I didnt have write access which git showed that It is enough to close a pull request. I tried by curiosity and It worked. I'll reopen it.

**Pull request merged**